content/en/docs/specs/status.md

```diff
@@ -92,7 +92,7 @@ same as the **Protocol** status.
 
 ### [Logging][]
 
-- {{% spec_status "Bridge API" "otel/logs/bridge-api" "Status" %}}
+- {{% spec_status "API" "otel/logs/api" "Status" %}}
```

Comment:
while there is a new concept in the spec called "Logs API", it's not stable, only the previous concept "Logs Bridge API" is stable

Reply:
Ok, I'll revert the changes I made to once again refer to "Bridge API".

Reply:
Reverted.